pkg/manager/member/ticdc_scaler.go

```diff
@@ -88,16 +95,20 @@ func (s *ticdcScaler) ScaleIn(meta metav1.Object, oldSet *apps.StatefulSet, newS
 	}
 	pod, err := s.deps.PodLister.Pods(ns).Get(podName)
 	if err != nil {
-		return fmt.Errorf("ticdcScaler.ScaleIn: failed to get pods %s for cluster %s/%s, error: %s", podName, ns, tcName, err)
+		return fmt.Errorf("ticdcScaler.ScaleIn: failed to get pods %s for cluster %s/%s, error: %v", podName, ns, tcName, err)
 	}
+	tc, _ := meta.(*v1alpha1.TidbCluster)
 
-	// when scaling in TiCDC pods, we let the "capture info" in PD's etcd to be deleted automatically when shutting down the TiCDC process or after TTL expired.
+	err = gracefulShutdownTiCDC(tc, s.deps.CDCControl, s.deps.PodControl, pod, ordinal, "ScaleIn")
+	if err != nil {
+		return err
```

Comment:
the log doesn't contain a stack about an error, so how about adding some context?

Reply:
More info added.